### ISSUE TITLE
Custom Sensors end the plugin on a reboot loop

### ISSUE THREAD

**Issue by DroidTester** [grzegorz914/homebridge-lgwebos-tv#214]
When creating input based sensors, the plugin ends up on a never-ending reboot loop till the custom sensor is disabled/deleted. 

have also got another issue, where the plugin logs shows updates to backlight/color levels, but nothing is reflected on the tv. requiring to manually adjust with the remote. TV is a LG C3 with the latest webos updates installed

**Comment by grzegorz914** [comment]
Log please

**Comment by DroidTester** [comment]
[homebridge-lgw.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15320689/homebridge-lgw.txt)
[homebridge-lgw-debug-on.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15320693/homebridge-lgw-debug-on.txt)

**Comment by grzegorz914** [comment]
fixed

**Comment by DroidTester** [comment]
Works perfectly now. Cheers mate ! 

the following is the log for the backlight issue I'd mentioned earlier as well. 
[backlight and color error.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15321142/backlight.and.color.error.txt)

**Comment by grzegorz914** [comment]
can you try 2.17.15-beta.0 and let me know wit the log

**Comment by DroidTester** [comment]
does not work with the new beta either, attaching the log

[2.17.15 beta log.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15321695/2.17.15.beta.log.txt)

**Comment by DroidTester** [comment]
with the updated beta plugin, both the color/backlight functionality seem to work. but produce a "lunaSend" popup on the tv screen that needs an "Okay" to be clicked for the action to take effect. attaching the new logs below 
[new beta log.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15323468/new.beta.log.txt)

**Comment by grzegorz914** [comment]
Nice, try next beta and get the log

**Comment by DroidTester** [comment]
the "lunaSend" pop up still exists and now with the beta13 it appears even when switching between hdmi inputs. 

[beta.13.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15324297/beta.13.txt)

**Comment by grzegorz914** [comment]
try beta.14 and get the log

**Comment by DroidTester** [comment]
no changes to report 
[beta.14.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15324546/beta.14.txt)

**Comment by grzegorz914** [comment]
try b.15

**Comment by DroidTester** [comment]
[beta.15.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15324751/beta.15.txt)

same issue persists

**Comment by grzegorz914** [comment]
try.17

**Comment by DroidTester** [comment]
[beta.17.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15325172/beta.17.txt)

both beta17 and 18 still have the issue

**Comment by grzegorz914** [comment]
can you make video of this behavior

**Comment by grzegorz914** [comment]
try 19

**Comment by DroidTester** [comment]
> can you make video of this behavior
have uploaded a small video over here - https://we.tl/t-fU3pYnBEGy

will test 19 now.

**Comment by DroidTester** [comment]
[beta.19..txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15325580/beta.19.txt)

device not detected with beta.19

**Comment by grzegorz914** [comment]
restart plugin again, I don't see this issue on my side

**Comment by DroidTester** [comment]
beta 19 was pulled before i reinstalled again, i had just checked the other webos plugin by merdok, the backlight & color sliders work on that somehow. 

however, i'm back with build beta.20 and its the same as the stable build 14. no "lunaSign" errors but no actual changes to the screen. only logs show the newly input values..  

[beta.20.txt](https://github.com/grzegorz914/homebridge-lgwebos-tv/files/15325807/beta.20.txt)

**Comment by DroidTester** [comment]
also, adding a bit of information i'm not sure is relevant or not. the LG C3 didnt have any option for the "lg connect apps" part of the plugin configuration and i was presented with a popup to allow mobile access only after enabling sslWebSocket. Post accepting, the plugin started working.
